gpt4all-chat/server.cpp

```diff
@@ -97,11 +97,12 @@ void Server::start()
             if (!MySettings::globalInstance()->serverChat())
                 return QHttpServerResponse(QHttpServerResponder::StatusCode::Unauthorized);
 
-            const QList<ModelInfo> modelList = ModelList::globalInstance()->exportModelList();
+            const QList<ModelInfo> modelList = ModelList::globalInstance()->userDefaultModelList();
```

Comment:
We can't call the function userDefaultModelList because as shown here, it is just listing the installed models, and is useful for more than just that combo box.

Reply:
Yes, this is why I included this FIXME:

```// FIXME: This needs to be kept in sync with m_selectableModels so should probably be merged```